zulipterminal/ui_tools/boxes.py

```diff
@@ -110,7 +110,7 @@ def autocomplete_mentions(self, text: str, state: int,
                               prefix_string: str) -> Optional[str]:
         # Handles user mentions (@ mentions and silent mentions)
         # and group mentions.
-        group_typeahead = ['@*{}*'.format(group_name)
+        group_typeahead = [prefix_string+'*{}*'.format(group_name)
```

Comment:
Group typeaheads don't support silent mentions I think?

Reply:
@kaustubh-nair Indeed, they don't. The `match_group(s)` function takes `text[1:]` as an argument which means that it'd never return `True` for silent mentions ('@_').